src/ext/java/org/opentripplanner/ext/siri/SiriAlertsUpdateHandler.java

```diff
@@ -199,7 +213,9 @@ private long getEpochSecond(ZonedDateTime startTime) {
   }
 
   /*
-   * Creates alert from PtSituation with all textual content
+   * Creates a builder for an internal model TransitAlert, including all textual content from the
+   * supplied SIRI PtSituation. The feed scoped ID of this TransitAlert will be the single feed ID
+   * associated with this update handler, plus the situation number provided in the SIRI feed.
```

Comment:
From a maintenance perspective, this may be a good place to document the structure of the IDs because they are created during this mapping operation. Although I value DRY, the same concern exists here as other places above: increasing visibility of docs for people reusing/configuring the encapsulated code might make it hard for the maintainer to find those descriptions and use them to quickly understand the actions of this method.

Reply:
I have also put the information in the class-level Javadoc, and rephrased the information for the two contexts. Also included a note that FeedScopedId creation strategy should be pluggable or configurable as a condition for moving out of sandbox.